features/step_definitions/logging.rb

```diff
@@ -1307,24 +1306,26 @@
   ensure_admin_tagged
   def check_log(logs, errors, exceptions)
     error_logs = []
-    logs.split("\n").each do | log |
-      if !(exceptions.nil?)
-        ignore = false
-        exceptions.each do | exception |
-          if log.include? exception
-            ignore = true
-            break
+    unless logs.empty? || logs.nil?
+      logs.split("\n").each do | log |
```

Comment:
The step `I record all pods logs in the "openshift-operators-redhat" project` generates a very long log list, we need a utilize for it.

Reply:
It would be better go and check the must gather logs, rather than print all of them in the CI.

Reply:
Logging is not deployed along with OCP, must-gather won't collect logging's resources.

Reply:
Would you mind to log an jira issue to me, so that I'll follow up to add logging logs along side (not as part of) with must gather logs.

Reply:
I'll remove the step from the scenario.

Reply:
Done